tests/fixtures/tumor-variant-freq-study-result.yaml

```diff
@@ -0,0 +1,97 @@
+id: cancer_hotspots:CIC-R215
+type: TumorVariantFrequencyStudyResult
+name: Overall Tumor Variant Frequencr for CIC R215
+sourceDataSet:
+  type: DataSet
+  name: Cancer Hotspots
+  version: "2"
+specifiedBy:
+  type: Method
+  reportedIn:
+    type: Document
+    doi: 10.1158/2159-8290.CD-17-0321
+    name: Accelerating Discovery of Functional Mutant Alleles in Cancer
+affectedTumorSamples: 25
+totalTumorSamples: 24592
+affectedFrequency: 0.001016590761223162
+focusVariant:
+  type: CategoricalVariant
+  name: CIC R215
+sampleGroup:
+  name: Overall
+  type: StudyGroup
+subGroupFrequency:
+  - type: TumorVariantFrequencyStudyResult
+    focusVariant:
+      type: CategoricalVariant
+      name: CIC R215W
+    affectedTumorSamples: 17
+    totalTumorSamples: 24592
+    affectedFrequency: 0.0006912817176317501
+    subGroupFrequency:
+      - type: TumorVariantFrequencyStudyResult
+        focusVariant:
+          name: CIC R215W
+          type: CategoricalVariant
+        sampleGroup:
+          name: CNS/Brain
```

Comment:
Consider using the `characteristics` attribute to capture a MappableConcept that formally describes the criteria restricting this sub-group.  e.g. as in the CAF Study Result example here: https://github.com/ga4gh/va-spec/blob/issue-319/tests/fixtures/cohort-allele-freq-test.yaml#L50.

Here we might say:

````
    studyGroup:
      id: remaining
      name: CNS/Brain Tumor SubGroup
      type: StudyGroup
      characteristics:
        conceptType: Disease
        name: CNS/Brain Tumors
````